pages/community/precice-workshop-2024.md

```diff
@@ -51,18 +57,105 @@ The cost of lunch, as well as coffee and snacks is included in the registration
+      <p>Zhen Wu, Sun Yat-sen University, China;</p>
+    </summary>
+    <p>In the field of geoscience, the full knowledge of atmospheric conditions is necessary in the fields of urban planning, disaster mitigation, emergency response and air pollution assessments. While we see the remarkable advantages of the meso-scale model of Weather Research and Forecast (WRF) in the simulation of the atmosphere, it shows significant deficiencies in describing the micro-scale effects of the complex underlying surface. For solving this problem, a new adapter is developed based on preCICE. Thanks to the open source nature of the WRF model, the developed adapter enables the WRF to be coupled with other widely used Computational Fluid Dynamics (CFD) software, such as OpenFOAM. In fact, the developed adapter aims to provide the WRF model the Fluid-Fluid coupling capacity, which could be used to conduct a trans-scale numerical simulation downscaling the results from a numeric weather prediction software to drive a micro-scale (~1m) flow simulation. Such a trans-scale simulation shows the detailed flow structures inside the urban boundary layer or above complex hilly terrain.</p>
+  </details>
+  <details class="workshop-event" id="talk-jingya-li">
+    <summary>
+      Isogeometric Analysis Suitable Coupling Methods for Fluid-Structure Interactions with Solid-solver G+Smo Coupled via preCICE<br/>
+      <p><a href="https://www.tudelft.nl/en/eemcs/the-faculty/departments/applied-mathematics/people/j-jingya-li-msc">Jingya Li</a>, Delft University of Technology, The Netherlands</p>
+    </summary>
+    <p>Accurate simulation of fluid-structure interactions (FSI) remains a difficult task in com- putational mechanics, especially when dealing with complicated geometries and dynamic coupling between fluid and solid domains. This paper introduces novel benchmarks in the field of FSI that take advantage of isogeometric analysis (IGA) and the adaptibility of the preCICE coupling library. We offer a framework that combines the IGA-based solid mechanics library G+Smo, the computational fluid dynamics capabilities of openFOAM, and the Julia-based WaterLily.jl fluid solver, aimed at advancing hydrodynamic simulations. Central to our approach is the utilization of spline-based communication for IGA-based fluid-structure interaction simulations. We employ spline-based communication instead of quadrature points to minimize the amount of information exchanged. A comparison of accuracy and efficiency between spline-based communication and quadrature point-based communication will be presented. Several benchmarks will be discussed, ranging from the replication of established preCICE cases to direct comparisons with other solid mechanics libraries, to demonstrate the effectiveness of spline-based communication. Through these benchmarks, we conclude that spline-based communication is more efficient than quadrature point-based communication and yields the same level of accuracy.
```

Comment:
Maybe there are more hyphenation issues in other abstracts, I did not check.

```suggestion
    <p>Accurate simulation of fluid-structure interactions (FSI) remains a difficult task in computational mechanics, especially when dealing with complicated geometries and dynamic coupling between fluid and solid domains. This paper introduces novel benchmarks in the field of FSI that take advantage of isogeometric analysis (IGA) and the adaptibility of the preCICE coupling library. We offer a framework that combines the IGA-based solid mechanics library G+Smo, the computational fluid dynamics capabilities of openFOAM, and the Julia-based WaterLily.jl fluid solver, aimed at advancing hydrodynamic simulations. Central to our approach is the utilization of spline-based communication for IGA-based fluid-structure interaction simulations. We employ spline-based communication instead of quadrature points to minimize the amount of information exchanged. A comparison of accuracy and efficiency between spline-based communication and quadrature point-based communication will be presented. Several benchmarks will be discussed, ranging from the replication of established preCICE cases to direct comparisons with other solid mechanics libraries, to demonstrate the effectiveness of spline-based communication. Through these benchmarks, we conclude that spline-based communication is more efficient than quadrature point-based communication and yields the same level of accuracy.
```